OpenSim/Framework/Communications/Services/LoginResponse.cs

```diff
@@ -400,6 +401,8 @@ public virtual XmlRpcResponse ToXmlRpcResponse()
                 responseData["region_y"] = (Int32)(RegionY * Constants.RegionSize);
 
                 responseData["map-server-url"] = MapServerURI;
+                responseData["profile-server-url"] = ProfileServerURI;
```

Comment:
This PR looks good except for these assignments, which should not be made if the string is empty or null, otherwise it triggers the viewer to use them.